securedrop/tests/conftest.py

```diff
@@ -160,10 +163,26 @@ def test_admin(journalist_app):
 def test_source(journalist_app):
     with journalist_app.app_context():
         source, codename = utils.db_helper.init_source()
-        filesystem_id = source.filesystem_id
+        utils.db_helper.submit(source, 2)
```

Comment:
My preference would be to split this out into a `submission` fixture so that it's not run on every call to `test_source`. This keeps tests faster and also doesn't have the (semi) unexpected behavior of a fixture providing multiple things.